website/templates/project/settings.mako

```diff
@@ -203,35 +201,64 @@
 
         % endif  ## End Select Addons
 
-        % if include_wiki_settings:  ## Begin Configure Wiki
-
-            % if not node['is_registration']:
-
-                <div class="panel panel-default">
-                    <span id="configureWikiAnchor" class="anchor"></span>
+        % if not node['is_registration']:   ## Begin Wiki Config
+            <div class="panel panel-default">
+                <span id="configureWikiAnchor" class="anchor"></span>
+                <div class="panel-heading clearfix">
+                    <h3 class="panel-title">Wiki</h3>
+                </div>
 
-                    <div class="panel-heading clearfix">
-                        <h3 class="panel-title">Configure Wiki</h3>
-                    </div>
-                    <div class="help-block" style="padding-left: 15px">
-                        <p class="text-info">These settings control who can edit your wiki. To make a wiki editable by all OSF users, make your project/component public.</p>
-                    </div>
-                    <form id="wikiSettings" class="osf-treebeard-minimal">
-                        <div id="wgrid">
-                            <div class="spinner-loading-wrapper">
-                                <div class="logo-spin logo-lg"></div>
-                                <p class="m-t-sm fg-load-message"> Loading wiki settings...  </p>
+            <div class="panel-body">
+                %if wiki:
+                    <form id="selectWikiForm">
+                        <div>
+                            <label>
+                                <input
+                                        type="checkbox"
+                                        name="${wiki.short_name}"
+                                        class="wiki-select"
+                                        data-bind="checked: enabled, click: onChange"
```

Comment:
Using `click` is OK, but I think using a subscriber function like in https://jsfiddle.net/rniemeyer/cnkVA/2/ might be better for a11y. There may be alternative ways to toggle the checkbox, such as tabbing and hitting Enter.

That said, this isn't a blocker. Let's go with the "5-minute rule"--if the change takes <= 5 mins to fix, go ahead and make it. Otherwise, defer.